core/src/mindustry/mod/ClassMap.java

```diff
@@ -459,6 +459,8 @@
         classes.put("DrawTurret", mindustry.world.draw.DrawTurret.class);
         classes.put("DrawWarmupRegion", mindustry.world.draw.DrawWarmupRegion.class);
         classes.put("DrawWeave", mindustry.world.draw.DrawWeave.class);
+        classes.put("ConsumeCoolant", mindustry.world.consumers.ConsumeCoolant.class);
+        classes.put("ConsumeLiquidFlammable", mindustry.world.consumers.ConsumeLiquidFlammable.class);
```

Comment:
What gradle command do I run to properly update the ClassMap?

Reply:
it IS updateScripts, but I believe something broke in that command.